app/models/spree/tax_rate.rb

```diff
@@ -21,7 +21,7 @@ class TaxRate < ApplicationRecord
 
     belongs_to :zone, class_name: "Spree::Zone", inverse_of: :tax_rates
     belongs_to :tax_category, class_name: "Spree::TaxCategory", inverse_of: :tax_rates
-    has_many :adjustments, as: :originator, dependent: :destroy
+    has_many :adjustments, as: :originator, dependent: nil
```

Comment:
I guess this is acknowledging that the default behaviour hasn't been a problem thus far.. but it seems to me it would be better to `:nullify`. That might require some code changes, which I don't suggest we do now. 
I'd rather just leave this (and all below) as `todo` until we need to change it.